airflow/providers/microsoft/azure/provider.yaml

```diff
@@ -81,9 +81,9 @@ dependencies:
   - adal>=1.2.7
   - azure-storage-file-datalake>=12.9.1
   - azure-kusto-data>=4.1.0
+  - azure-mgmt-containerinstance>=7.0.0,<9.0.0
   # TODO: upgrade to newer versions of all the below libraries.
   #   See issue https://github.com/apache/airflow/issues/30199
-  - azure-mgmt-containerinstance>=1.5.0,<2.0
```

Comment:
Lets keep it under the TODO as we should also bump it to be compatible with >=9.0